python/dgl/nn/pytorch/explain/gnnexplainer.py

```diff
@@ -0,0 +1,412 @@
+"""Torch Module for GNNExplainer"""
+# pylint: disable= no-member, arguments-differ, invalid-name
+from math import sqrt
+import torch
+
+from torch import nn
+from tqdm import tqdm
+
+from ....base import NID, EID
+from ....subgraph import khop_in_subgraph
+
+class GNNExplainer(nn.Module):
+    r"""
+
+    Description
+    -----------
+    GNNExplainer model from paper `GNNExplainer: Generating Explanations for
+    Graph Neural Networks <https://arxiv.org/abs/1903.03894>`__ for identifying
+    compact subgraph structures and small subsets of node features that play a
+    critical role in GNN-based node classification and graph classification.
+
+    Parameters
+    ----------
+    model : nn.Module
+        The GNN model to explain.
+
+        * The required arguments of its forward function are graph and feat.
+          The latter one is for input node features.
+        * It should also optionally take an eweight argument for edge weights
```

Comment:
But eweight is also required right? Does it have to be a keyword argument or positional argument as the 3rd?

Reply:
You can write a simple code to described to interface